rpc/src/v1/impls/ibc.rs

```diff
@@ -102,21 +102,29 @@ where
     }
```

Comment:
For the commit message "Fix to provide correct seal and validator set", please write what was the problem and how you fixed it.